samples/graph-notification-team-channel/csharp/GraphNotification/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "MicrosoftAppId": "<< App ID>>",
+  "MicrosoftAppPassword": "<< Client Secret >>",
+  "MicrosoftAppTenantId": "<< Tenant ID >>",
+  "BaseUrl": "<< Ngrok >>",
+  "Base64EncodedCertificate": "<< Certificate >>",
+  "EncryptionCertificateId": "Notification",
+  "CertificateThumbprint": "<< Thumbprint >>"
```

Comment:
Why do we need these ?

Reply:
As give in the Sample Doc snippet 

![image](https://user-images.githubusercontent.com/109147706/186930822-4abda896-c65e-4dd9-878b-aab47f618a3e.png)